src/main/java/mate/academy/bookstore/repository/BookRepositoryImpl.java

```diff
@@ -42,6 +43,16 @@ public Book save(Book book) {
         return book;
     }
 
+    @Override
+    public Optional<Book> findById(Long id) {
+        try (Session session = sessionFactory.openSession()) {
+            Book book = session.get(Book.class, id);
+            return Optional.ofNullable(book);
```

Comment:
You don't need to create an extra variable - return Optional.ofNullable(session.get(Book.class, id));